client/src/components/NewReviewForm.js

```diff
@@ -0,0 +1,116 @@
+import React, {useState } from "react"
+import ErrorList from "./layout/ErrorList"
+
+const NewReviewForm = ({ pizzaId, pizzaPlace, setPizzaPlace, currentUser }) => {
+  const [newReview, setNewReview] = useState({
+    userId: currentUser.id,
+    rating: 3,
+    title: "",
+    text: ""
+  })
+
+  const [errors, setErrors] = useState({})
+
+  const postNewReview = async(newReviewData) => {
+    try {
+      const response = await fetch(`/api/v1/pizza-places/${pizzaId}/reviews/new`, {
```

Comment:
proper path here would be `/api/v1/pizza-places/${pizzaId}/reviews` - the `/new` at the end is not proper pathing when following RESTful applications and considering the HTTP verbs and CRUD actions

https://www.restapitutorial.com/lessons/httpmethods.html

Remember that the paths we defined for the user to navigate to in the application is different, so we could have the user visit this form from a `/pizza-places/:id/reviews/new` path, but we want to use the above for making the fetch request